operations/rack_resiliency/manage_critical_services.md

```diff
@@ -0,0 +1,404 @@
+# Editing/Deleting critical services from static ConfigMap (`rrs-mon-static`)
+
+## Overview
+The `rrs-mon-static` ConfigMap in the `rack-resiliency` namespace contains configurations for critical services.
+
+**1 Procedure to delete a critical service from the ConfigMap**
+
+1.1 Verify that the ConfigMap exists:
+
+```bash
+kubectl get ConfigMaps -n rack-resiliency
+```
+
+**Example output:**
+
+```text
+NAME                 DATA   AGE
+istio-ca-root-cert   1      6d21h
+kube-root-ca.crt     1      6d21h
+rrs-mon-dynamic      2      3d4h
+rrs-mon-static       11     3d4h
+```
+
+Here our list of critical services are present in `rrs-mon-static`.
+
+**To verify if the list is present: **
```

Comment:
This is coming out with `**`, not in bold, maybe getting rid of the space will fix it?
```suggestion
**To verify if the list is present:**
```